plugins/maven/graphql-java-codegen-maven-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/GraphQLCodegenMojo.java

```diff
@@ -248,6 +249,9 @@ public class GraphQLCodegenMojo extends AbstractMojo implements GraphQLCodegenCo
     @Parameter(defaultValue = "false")
     private boolean skip;
 
+    @Parameter(defaultValue = "true")
```

Comment:
Please use `MappingConfigConstants.DEFAULT_` for the default value of this parameter (for consistency purposes).